src/client/src/routes/PublicationsPage/components/PublicationsPage.js

```diff
@@ -5,8 +5,8 @@ import { arrayToObject } from "../../../resources/methods";
 import Input from '../../../components/form/Input';
 
 const PublicationsPage = (props) => {
-  const { userId } = props;
-
+  // const { userId } = props;
+  const userId = '123456789';
```

Comment:
why is the userId hardcoded here?

Reply:
we need to create a component that holds all the forms information and also sets the userId in the sessionstorage that it recieves from the db. This will be implemented in the following sprint